src/Compilers/Test/Utilities/CSharp/BasicCompilationUtils.cs

```diff
@@ -32,7 +33,7 @@ private static VisualBasicCompilation CreateCompilationWithMscorlib(string sourc
             {
                 assemblyName = TestBase.GetUniqueName();
             }
-            var tree = VisualBasicSyntaxTree.ParseText(source);
+            var tree = VisualBasicSyntaxTree.ParseText(SourceText.From(source, encoding: null, SourceHashAlgorithm.Sha256));
```

Comment:
```suggestion
            var tree = VisualBasicSyntaxTree.ParseText(SourceText.From(source, encoding: null, SourceHashAlgorithmUtils.DefaultContentHashAlgorithm));
```


Will hold off adding more of these suggestions until you confirm you agree with it.

Reply:
I'm fine updating tests that do not care about the specific alg to use the default. Some tests need to user a particular one though.